### DISCUSSION TITLE
How to reverify education discount without a card?

### DISCUSSION THREAD

**Discussion by phpeterson-usf** [community/community#31399] [GitHub Education]
I've been using GitHub Classroom for a couple semesters, and I see that I need to reverify my education discount. However, it appears that the only way to do that is using my laptop's camera to take a picture of an ID card. We don't have ID cards anymore, and they never had expiration dates on them anyway. Is there a link I can use to upload another form of proof? The web page seems only to have the camera functionality. Thanks for any tips

Phil Peterson 
University of San Francisco

**Comment by JosafatJimenezB** [comment]
I didn't have a credential anyway, so what I did was to send a picture of my school schedule showing the date that covered the semester and a picture showing that I was enrolled all together in one picture.

**Reply by phpeterson-usf** [reply]
Thanks but how did you send it? The [upload proof page](https://education.github.com/discount_requests/8195912/additional_information) seems to only support photos from the camera

**Reply by JosafatJimenezB** [reply]
At the bottom left of the example image it says about uploading an image, there you click on it and it will let you upload a .jpg or .png image.
